src/main/java/it/gov/pagopa/gpd/ingestion/manager/service/impl/PDVTokenizerServiceRetryWrapperImpl.java

```diff
@@ -1,67 +1,32 @@
 package it.gov.pagopa.gpd.ingestion.manager.service.impl;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
-import io.github.resilience4j.core.IntervalFunction;
-import io.github.resilience4j.core.functions.CheckedFunction;
-import io.github.resilience4j.retry.Retry;
-import io.github.resilience4j.retry.RetryConfig;
-import io.github.resilience4j.retry.RetryRegistry;
+import io.github.resilience4j.retry.annotation.Retry;
 import it.gov.pagopa.gpd.ingestion.manager.exception.PDVTokenizerException;
-import it.gov.pagopa.gpd.ingestion.manager.exception.PDVTokenizerUnexpectedException;
 import it.gov.pagopa.gpd.ingestion.manager.service.PDVTokenizerService;
 import it.gov.pagopa.gpd.ingestion.manager.service.PDVTokenizerServiceRetryWrapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 /**
  * {@inheritDoc}
  */
+@Service
 public class PDVTokenizerServiceRetryWrapperImpl implements PDVTokenizerServiceRetryWrapper {
 
-    private static final Long INITIAL_INTERVAL = Long.parseLong(System.getenv().getOrDefault("PDV_TOKENIZER_INITIAL_INTERVAL", "200"));
-    private static final Double MULTIPLIER = Double.parseDouble(System.getenv().getOrDefault("PDV_TOKENIZER_MULTIPLIER", "2.0"));
-    private static final Double RANDOMIZATION_FACTOR = Double.parseDouble(System.getenv().getOrDefault("PDV_TOKENIZER_RANDOMIZATION_FACTOR", "0.6"));
-    private static final Integer MAX_RETRIES = Integer.parseInt(System.getenv().getOrDefault("PDV_TOKENIZER_MAX_RETRIES", "3"));
-
     private final PDVTokenizerService pdvTokenizerService;
-    private final Retry retry;
 
-    PDVTokenizerServiceRetryWrapperImpl(PDVTokenizerService pdvTokenizerService, Retry retry) {
+    @Autowired
+    PDVTokenizerServiceRetryWrapperImpl(PDVTokenizerService pdvTokenizerService) {
         this.pdvTokenizerService = pdvTokenizerService;
-        this.retry = retry;
-    }
-
-    public PDVTokenizerServiceRetryWrapperImpl() {
-        RetryConfig config = RetryConfig.custom()
```

Comment:
some of this configuration are missing in `application.properties` consider creating a `Retry` bean with this configuration as some of the properties like `retryOnException` and `intervalFunction` seems to be not replicable there